src/nimblepkg/common.nim

```diff
@@ -64,3 +63,15 @@ when not defined(nimscript):
 
 const
   nimbleVersion* = "0.10.2"
+
+when not declared(initHashSet):
+  import sets
+
+  template initHashSet*[A](initialSize = 64): HashSet[A] =
+    initSet[A](initialSize)
+
+when not declared(toHashSet):
+  import sets
```

Comment:
This might fail if the above ``import`` is present too, have you checked whether that's the case?

Reply:
I tested that there is not an error, nor a warning, when a used module is being imported multiple times.